lib/ansible/cli/doc.py

```diff
@@ -473,8 +473,12 @@ def add_fields(text, fields, limit, opt_indent):
 
             if isinstance(opt['description'], list):
                 for entry in opt['description']:
+                    if not isinstance(entry, string_types):
+                        raise AnsibleError("Expected string in %s description, got %s" % (o, type(entry)))
```

Comment:
since it is a list, you can point to the specific element in description that is a problem

Reply:
`o` contains name of the element which has a problem.

Reply:
its a list of elements, i mean the index of that list:
```
option:
  description:
    - one ...
    - two ...
    - three: is the error
```